README.md

```diff
@@ -112,7 +112,7 @@ For more examples check out our [LLM Inference Guide](https://docs.openvino.ai/2
 optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code --weight-format fp16 MiniCPM-V-2_6
 
 #(Recommended) Same as above but with compression: language model is compressed to int4, other model components are compressed to int8
-optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code --weight-format int4 MiniCPM-V-2_6
+optimum-cli export openvino --model openbmb/MiniCPM-V-2_6 --trust-remote-code --weight-format int4 --group-size 16 MiniCPM-V-2_6
```

Comment:
The minimum group size we committed to support is 32. @helena-intel, can it work with group size of 32, or 64? The higher is better for performance and model footprint.

Reply:
Divisors of channel size 4304 are 1, 2, 4, 8, 16, 269, 538, 1076, 2152, 4304 so 32 and 64 don't work either. I tried channel-wise quantization (group size -1), but with the example script the output was empty (the same script worked fine with group size 16). If group size 16 is not supported, should we remove the INT4 recommendation?

Reply:
Thanks @helena-intel. Actually, the issue is related to the fact that we compress Vision model to int4 as well which is unexpected. We will prepare a fix for Optimum-Intel. But I would use `int8` for now.

Reply:
As an alternative you can use dataset `--weight-format int4 --dataset contextual`. It works on my end.

Reply:
Thank you, that is great!